lib/project_types/script/layers/domain/push_package.rb

```diff
@@ -10,7 +10,9 @@ class PushPackage
           :script_json,
           :script_content,
           :compiled_type,
-          :metadata
+          :metadata,
+          :library_language,
+          :library_version
```

Comment:
Just curious what your thoughts are on combining these into a single `Library` object instead of enumerating the fields. I'm just thinking that soon we'll probably add another argument `library_name`, and it feels like grouping them together (like in the GraphQL API) makes the most sense long-term. WDYT?

Reply:
Grouping it up makes sense! 👍  I've gone ahead and just made it a hash for now.